subprojects/ide/src/integTest/groovy/org/gradle/plugins/ide/AbstractSourcesAndJavadocJarsIntegrationTest.groovy

```diff
@@ -484,6 +484,31 @@ dependencies {
         ideFileContainsNoSourcesAndJavadocEntry()
     }
 
+    def "does not add project repository to download localGroovy() sources"() {
```

Comment:
I was a bit too quick in my review and I am pretty sure this will fail to merge.
The test needs an `@ToBeFixedForConfigurationCache`

Reply:
Ok, I'll add the annotation. Which task should I run to catch such things?

Reply:
Just pushed an update with this annotation, sorry for delay

Reply:
Sorry, I missed your question. The task is `configCacheIntegTest`

Reply:
Thanks